pkg/secrethub/http/options.go

```diff
@@ -0,0 +1,32 @@
+package http
+
+import (
+	"net/http"
+	"time"
+)
+
+type Option func(*Client)
```

Comment:
Sweet! 😍 Needs comments though.

Reply:
Also, any specific reason for calling this `Option` as opposed to `ClientOption` which we also use in the `secrethub` package?

Reply:
Renamed to `ClientOption`.